addons/mail/static/src/components/call_view/call_view.js

```diff
@@ -110,7 +109,7 @@ export class CallView extends Component {
         });
 
         this.state.tileWidth = tileWidth;
-        this.state.tileHeight = tileHeight;
+        this.callView.update({ tileHeight });
```

Comment:
You are using the private update for the with and the public update for the height.
You should use the public method if it's possible.

Reply:
This one in this PR is correct. The other PR that uses `_update()` is wrong.